staging-manifest.yml

```diff
@@ -10,6 +10,7 @@ applications:
   env:
     RAILS_ENV: staging
     RACK_ENV: staging
+    CKAN_REDIRECTION_URL: https://ckan.staging.publishing.service.gov.uk/
```

Comment:
I don't think we need the trailing slash, otherwise the user will be redirected to `https://ckan.staging.publishing.service.gov.uk///somewhere` (the slash is already defined on line 12 of `app/controllers/pages_controller.rb`).